ai-engineering/quickstart.mdx

```diff
@@ -214,7 +216,12 @@ ANTHROPIC_API_KEY=""
 Enter the API keys for the LLMs you want to work with.
 </Info>
 
+<Note>
+To run evaluations, you’ll need to authenticate the Axiom CLI. See [Setup and authentication](/ai-engineering/evaluate/setup) for details on using `axiom auth login`.
+</Note>
```

Comment:
This is more for the dev team i guess, but we absolutely need to allow "true offline" evals that just report to console or a file, and don't require auth.